pupil_src/shared_modules/video_capture/utils.py

```diff
@@ -10,8 +10,52 @@
 '''
 
 import numpy as np
+import cv2
 
 
+class GetExposureTime(object):
+    def __init__(self, max_ET, frame_rate, mode="manual"):
+        self.mode = mode
+        self.ET_thres = 1, min(10000 / frame_rate, max_ET)
+        self.last_ET = self.ET_thres[1]
+
+        self.targetY_thres = 90, 150
+
+        self.AE_Win = np.array([[3, 1, 1, 1, 1, 1, 1, 3],
+                                [3, 1, 1, 1, 1, 1, 1, 3],
+                                [2, 1, 1, 1, 1, 1, 1, 2],
+                                [2, 1, 1, 1, 1, 1, 1, 2],
+                                [2, 1, 1, 1, 1, 1, 1, 2],
+                                [2, 1, 1, 1, 1, 1, 1, 2],
+                                [3, 1, 1, 1, 1, 1, 1, 3],
+                                [3, 1, 1, 1, 1, 1, 1, 3]])
+        self.smooth = 1/3
+        self.check_freq = 0.1/3
+        self.last_check_timestamp = None
+
+    def cal_exposure_time(self, frame):
```

Comment:
I would like to refactor the name to `calculate_based_on_frame()`. This would [avoid redundant labeling](https://gist.github.com/sloria/7001839#general-naming-guidelines).